packages/paste-core/components/breadcrumb/src/index.tsx

```diff
@@ -14,39 +15,51 @@ const BreadcrumbSeparator: React.FC = () => (
     paddingLeft="space20"
     paddingRight="space20"
     role="presentation"
+    element={`${element}_SEPARATOR`}
```

Comment:
Since this component isn't being exported, and is directly passed from BreadcrumbItem, you only need to pass `element={element}`

Reply:
So you are saying we should move the `_SEPARATOR` and the string interp to line 62?

Reply:
Yup